packages/editor-ui/src/components/ExecutionsList.vue

```diff
@@ -285,7 +301,8 @@ export default mixins(externalHooks, genericHelpers, executionHelpers, restApi,
 				finishedExecutionsCount: 0,
 				finishedExecutionsCountEstimated: false,
 
-				checkAll: false,
+				allVisibleSelected: false,
```

Comment:
this is the problem that I shared.. we should not have a flag for this.. it should be a set of ids instead..

Reply:
I added it because when all existing is selected there is a  filter prop (`deleteBefore`) created to tell the server which should be deleted.
In this case there are no IDs added to the`selectedItems` (however we could do that in some cases that could be too much)